bin/GameIndex.dbf

```diff
@@ -11993,6 +11993,13 @@ Serial = SLES-52418
 Name   = Powerdrome
 Region = PAL-M5
 Compat = 4
+[patches = 52085DF1]
+	//Game takes advantage of the VU's extended float range
+	//This limits it so x86 can handle it
+	comment=VU Float patch by refraction
```

Comment:
`author = refraction` would be better
Comments can be slightly better formatted, add spaces after // and periods at end (ocd satisfaction).